addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -506,7 +508,24 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
                 }
             }
         }
-    }
+    },
+    /**
+     * @private
+     * @param {OdooEvent} ev
+     * @param {string[]} ev.data.fontsToLoad
+     * @param {Promise} ev.data.onSuccess
+     * @param {Promise} ev.data.onFailure
+     */
+    _onLoadFonts(ev) {
+        // TODO ideally, remove the <link> elements created once this widget
+        // instance is destroyed (although it should not hurt to keep them for
+        // the whole backend lifecycle).
+        const proms = ev.data.fontsToLoad.map(async fontURL => loadCSS(fontURL));
+        Promise.all(proms).then(
+            () => ev.data.onSuccess(),
+            () => ev.data.onFailure()
+        );
+    },
```

Comment:
Meh, in fact I don't see why you needed to split the logic over here, the options can load its font using loadCSS by itself... :grimacing: